__init__.py

```diff
@@ -204,12 +204,13 @@ def change_pronouns(self, reminder):
         Arguments:
             reminder (str): reminder text
         """
-        my_regex = r'\b{}\b'.format(self.translate('My'))
-        our_regex = r'\b{}\b'.format(self.translate('Our'))
-        your_word = self.translate('Your')
-        reminder = re.sub(my_regex, your_word, reminder)
-        reminder = re.sub(our_regex, your_word, reminder)
-        return reminder
+        vocabulary = self.translate_namedvalues("Perspective")
+
+        reminder = f" {reminder}"
```

Comment:
Yeah the space at the end seems like a good change. Shouldn't happen in english unless the stt is cut off but could in some other language.